src/components/ChatContent/Messages/IrcMessage.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <message
+    @mouseleave="hover = false"
+    @mouseover="hover = true"
+    v-show="$store.state.ignoredUsers.indexOf(message.user.username) === -1"
```

Comment:
If so, then when we change it so banned users have <message deleted> instead of removing the message, would you put the logic of choosing between a real message and an ignored message in the tab or in the message?

Reply:
I'd put it in the tab, especially since
1) the message will probably look substantially different
2) if there's multiple in a row, they should be "collapsed" into a single message
3) this should be a v-if, not a v-show (it's not constantly changing)

Reply:
![image](https://user-images.githubusercontent.com/6932278/61364683-ff168880-a88e-11e9-9c98-b28eed47569f.png)
Regardless of this error, since you said that in the end we'll merge ignored messages, I think it would make sense to do this in the getter, even though I think it's weird to have the decision between the ignored message comp and the normal message comp happen in vuex...

Reply:
I changed this to loop over getters.unignoredMessages, instead of putting the conditional in the html.

Reply:
What you probably want instead is:
```vue
<template v-for="...[getter that has ignored messages merged]">
  <message v-if="...">
  <ignoredthingie v-else>
</template>
```
Or something like that? And for now, just put one thing inside the template?